src/ui/components/pages/assets/moreActions.tsx

```diff
@@ -1,20 +1,22 @@
 import * as React from 'react';
 import * as styles from './moreActions.module.css';
-import cn from 'classnames';
 
 export function MoreActions({ children }: { children: React.ReactNode }) {
   const [isExpanded, setIsExpanded] = React.useState(false);
 
   return (
     <div
-      className={cn(styles.moreActions, isExpanded && styles.expanded)}
+      className={styles.moreActions}
       onMouseLeave={() => setIsExpanded(false)}
     >
-      {isExpanded &&
-        React.Children.map(
-          children,
-          child => child && React.cloneElement(child as React.ReactElement)
-        )}
+      {isExpanded && (
+        <div className={styles.content}>
+          {React.Children.map(
+            children,
+            child => child && React.cloneElement(child as React.ReactElement)
```

Comment:
Ow.. There were several mouse event handlers for hover action buttons.
Later I figured out how it can be done with pure css and forgot to remove that part.
So it can be replaced.